pkg/tests/api/alerting/api_ruler_test.go

```diff
@@ -240,15 +309,18 @@ func TestIntegrationAlertRulePermissions(t *testing.T) {
 	})
 
 	t.Run("when permissions for folder2 removed", func(t *testing.T) {
-		// remove permissions from folder2
+		// remove permissions for folder2
 		removeFolderPermission(t, permissionsStore, 1, userID, org.RoleEditor, "folder2")
+		// remove permissions for subfolder (inherits from folder1)
+		removeFolderPermission(t, permissionsStore, 1, userID, org.RoleEditor, "subfolder")
```

Comment:
I think this is ok but I would really love to see it as a separate sub-test.

Reply:
done in 060b541f67b86ff77480680d05c05dd03045d7f2